pyatlas/package.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+# THIS ENTIRE SCRIPT IS A MASSIVE HACK
+# THIS SHOULD REALLY BE DONE WITH GRADLE
```

Comment:
amazing :)

Reply:
I really need to remove this comment... :P